game.py

```diff
@@ -0,0 +1,90 @@
+from etc.utils import Color, read_words, get_random_word, is_valid_word
+from collections import defaultdict
+from etc.consts import MAX_ATTEMPTS, INDEX_NOT_FOUND
+
+
+class WordleGame:
+    words = read_words()
+
+    def __init__(self):
+        self.__goal_word = ""
+        self.attempts = 0
+        self.word_found = False
+        self.reset()
+
+    def apply_answer(self, guess):
+        result = None
+
+        if not is_valid_word(guess):
```

Comment:
When you have a flow like:
```py
if something:
    # code
    return  # or exception
else:
    # more code
```
You can simply remove the `else`, and make the rest of your code unindented - since if the condition is true, the function will exit before it ever reaches your code.
```py
if something:
    # code
    return  # or exception
# more code
```